src/main/java/core/basesyntax/model/User.java

```diff
@@ -8,6 +8,12 @@ public class User {
     private String password;
     private Integer age;
 
+    public User(String login, String password, Integer age) {
+        this.login = login;
+        this.password = password;
+        this.age = age;
+    }
```

Comment:
I think this is a bad practice, in my opinion in real projects you wouldn't have access to change this class just to write tests. You have set and get methods for this.